superset-frontend/packages/superset-ui-demo/storybook/stories/plugins/legacy-preset-chart-deckgl/Arc/Stories.tsx

```diff
@@ -0,0 +1,101 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+/* eslint-disable sort-keys */
+/* eslint-disable no-magic-numbers */
+import React from 'react';
+import { SuperChart } from '@superset-ui/core';
+import { ArcChartPlugin } from '@superset-ui/legacy-preset-chart-deckgl';
+import payload from './payload';
+import dummyDatasource from '../../../../shared/dummyDatasource';
```

Comment:
NIT: can we simplify this import? Maybe by setting a base path in `tsconfig.json` to `storybook`? Then we could import like `import dummyDatasource from 'storybook/shared/dummyDatasource';` WDYT?

Reply:
(comment applies to other files below that import dummyDatasource like that)

Reply:
The `absolute import` depends on how to locate the `storybook/shared` module. (Maybe needed module alias in somewhere).

so `relative import` is straightforward here.